agent-chat-assistant/config.yml

```diff
@@ -9,67 +9,101 @@ steps:
     This tutorial walks through the steps to create an agent assistant for chat interactions. An agent assistant will display suggested responses based on predefined keywords and responses.
 - title: Configure SDK Settings
   content: |
-    Next is the configuration of SDK setting by providing an access token which will be used in processing API calls.
+    First is the configuration of SDK setting by providing an access token which will be used in processing API calls.
+
     To obtain a token, separate function is created to request token from PureCloud.
 - title: Obtain an Access Token
   content: |
     Start by making a POST call to https://login.mypurecloud.com/oauth/token to request a token. Some SDKs will include helper functions to obtain the token using the Client ID and Client Secret, while on other SDKs, the HTTP Request and Response needs to be manually handled to obtain the token.
+    
     For this tutorial, Implicit Grant Type will be used as user context is not needed in acquiring the desired information.
+    
     More information about OAuth and Authorization could be found in the [documentation](https://developer.mypurecloud.com/api/rest/authorization/)
 - title: Instantiate APIs
   content: |
     Instance of User API, Conversation API and Notification API will be created and will be passed to different methods.
 - title: Get User Details
   content: |
     Before getting the conversations, we need to get the logged in user's details. We will then call the getUsersMe function of the Users API to get the userId.
+    
+    Once we get the user details, we will call the setupChatChannel function.
 - title: Setup Chat Channel
   content: |
-    The setupChatChannel function will be called to subscribe to incoming chat conversations. This function handles the events when a chat conversation fires.
+    The setupChatChannel function will be called to subscribe to incoming chat conversations. This function triggers the onChatConversationEvent function.
 - title: Create a Channel in the setupChatChannel function
   content: |
-    In the setupChatChannel function, we will first create a channel. The Notifications API will be called to post a notifications channel. We will also create a web socket to handle the on socket messages.
+    In the setupChatChannel function, we will first create a channel. The postNotificationsChannels of Notifications API will be called. We will also create a web socket to handle the on socket messages.
```

Comment:
Missing the word "function" before "of"

Reply:
Thanks Tim, updated the file accordingly.